### PR TITLE
Instructional updates

### PR DESCRIPTION
**Pull request by joinimran**
As suggested, some of the information was missing and has been added.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8567

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by illfated** [comment]
@joinimran : The changes look good to me. Would you be able to edit the file, to change a sentence 2 lines above the current changes? (this is out of reach for suggestions by us other users in this PR) The sentence I am thinking of, is this one:
```
1. Log on to [Azure](https://portal.azure.com) with user that has **Global Administrator** role.
```
I would like to suggest changing "user" to "**a user account**", if possible.

Don't worry, if you find it difficult to edit the file, let's just declare it "out of scope" for this PR.

**Comment by joinimran** [comment]
@illfated - Noted and applied.

File: windows/security/threat-protection/microsoft-defender-atp/exposed-apis-create-app-nativeapp.md [hunk 1]
```diff
@@ -50,18 +50,30 @@ This page explains how to create an AAD application, get an access token to Micr
 
 ## Create an app
 
-1. Log on to [Azure](https://portal.azure.com) with user that has **Global Administrator** role.
+1. Log on to [Azure](https://portal.azure.com) with user account that has **Global Administrator** role.
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/exposed-apis-create-app-nativeapp.md:53]: improvement suggestion:
```suggestion
1. Log on to [Azure](https://portal.azure.com) with a user account that has the **Global Administrator** role.
```

**Comment by illfated** [comment]
If others want to proofread the PR content, the Rich diff may be easier to browse (even though the images extends the page a bit), this is the direct link (in case you aren't familiar with the feature button in the source view):
- https://github.com/MicrosoftDocs/windows-itpro-docs/pull/8598/files?short_path=82073fd#diff-82073fd718b5610c43aaf68c76cb488897546da95b089ddd4eedd1f5c2850d73

**Review by illfated — APPROVED** [review]
Looks good to me, but I might have missed something.

**Comment by nam31** [comment]
@mjcaparas PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**